nginx_config_reloader/__init__.py

```diff
@@ -78,8 +79,19 @@ def process_IN_MOVE_SELF(self, event):
         raise ListenTargetTerminated
 
     def handle_event(self, event):
+        queue_length = len(self.notifier._eventq)
+        if queue_length > 1:
+            self.logger.info("Detected other reloads in queue, skipping this one.")
+            return
         if not any(fnmatch.fnmatch(event.name, pat) for pat in WATCH_IGNORE_FILES):
-            self.logger.info("{} detected on {}".format(event.maskname, event.name))
+            self.logger.info(
+                "{} detected on {}. Reloading in {} seconds".format(event.maskname, event.name, HANDLE_SLEEP)
+            )
+            time.sleep(HANDLE_SLEEP)
```

Comment:
we're sleeping in the middle of the event handler, seems a bit self-crafted, couldn't we use the notifier itsel?
it provides options to control how fast it reads events:
https://github.com/seb-m/pyinotify/blob/0f3f8950d12e4a6534320153eed1a90a778da4ae/python3/pyinotify.py#L1100